requirements.txt

```diff
@@ -5,3 +5,4 @@ quart
 quart-cors
 python-dotenv
 hypercorn
+matplotlib
```

Comment:
oops, actually that one also has a dependency on matplotlib:

https://github.com/pratiman-91/colormaps/blob/8f96cd26fc4e18c2bc4fcfbc0c90b73e3048c228/setup.py#L138

Reply:
instead of adding a multimegabyte dependency, we might as well just have a constant for the palette since it's a simple array of hex / rgb values:

You'll like this one: https://github.com/karthik/wesanderson

there's also a palette listed here: http://www.cookbook-r.com/Graphs/Colors_(ggplot2)/#a-colorblind-friendly-palette

or copy set1:

https://github.com/matplotlib/matplotlib/blob/3b9b1a222a39190695028271fbce8de90f963c1d/lib/matplotlib/_cm.py#L919

Reply:
Change it

Reply:
Oh I'm sold with Wes Anderson 🎥 !

Reply:
Only issue with Wes and GGplot is they are for the R language, so I copy the set1 palette